tests/test_mathematical.py

```diff
@@ -1445,6 +1445,40 @@ def test_power_scalar(shape, dtype):
     assert_allclose(result, expected, rtol=1e-6)
 
 
+class TestNanToNum:
```

Comment:
please add tests to `test_sycl_queue` and `test_usm_type` scope, specially with `copy=True` use case.